src/util/string_utils.cpp

```diff
@@ -162,17 +162,19 @@ std::string escape(const std::string &s)
 
 /// Replace all occurrences of a string inside a string
 /// \param [out] str: string to search
-/// \param from: string to replace
+/// \param from: string to replace; must be non-empty
 /// \param to: string to replace with
 /// Copyright notice:
 /// Attributed to Gauthier Boaglio
 /// Source: https://stackoverflow.com/a/24315631/7501486
 /// Used under MIT license
+/// Note that this has quadratic complexity if from.size() != to.size()
```

Comment:
Quadratic in ... what? The length of `str`? That's equally true then of the implementation that triggered this discussion in #3463.